packages/mattermost-redux/src/client/client4.ts

```diff
@@ -3837,13 +3834,22 @@ export default class Client4 {
 
     // Client Helpers
 
-    doFetch = async <T>(url: string, options: Options): Promise<T> => {
-        const {data} = await this.doFetchWithResponse<T>(url, options);
+    doFetch = async <ClientDataResponse>(url: string, options: Options): Promise<ClientDataResponse> => {
+        const {data} = await this.doFetchWithResponse<ClientDataResponse>(url, options);
 
         return data;
     };
 
-    doFetchWithResponse = async <T>(url: string, options: Options): Promise<ClientResponse<T>> => {
+    /**
+     * @param query string query of graphQL, pass the json stringified version of the query
+     * eg.  const query = JSON.stringify({query: `{license, config}`});
+     *      client4.doFetchWithGraphQL(query);
+     */
+    doFetchWithGraphQL = async <DataResponse>(query: string) => {
```

Comment:
The existing `doFetch` methods were intended to be private, but we didn't have `private/public` modifiers at the time since it was pre-TypeScript. Usually I use `doX` naming for helper methods. 

Could you actually name this to just `fetchWithGraphQL` and move it to be above the `// Client Helpers` comment above?